test/timing_load_creds.c

```diff
@@ -0,0 +1,174 @@
+/*
+ * Copyright 2020-2022 The OpenSSL Project Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License 2.0 (the "License").  You may not use
+ * this file except in compliance with the License.  You can obtain a copy
+ * in the file LICENSE in the source distribution or at
+ * https://www.openssl.org/source/license.html
+ */
+
+#include <stdio.h>
+#include <stdlib.h>
+
+#ifndef _WIN32
+# include <sys/stat.h>
+# include <openssl/pem.h>
+# include <openssl/x509.h>
+# include <openssl/err.h>
+# include <openssl/bio.h>
+# include <../include/internal/e_os.h>
+# if defined(OPENSSL_SYS_UNIX)
+#  include <sys/resource.h>
+# endif
+
+static char *prog;
+
+static void readx509(const char *contents, int size)
+{
+    X509 *x = NULL;
+    BIO *b = BIO_new_mem_buf(contents, size);
+
+    if (b == NULL)
+        ERR_print_errors_fp(stderr), exit(1);
```

Comment:
Shouldn't these, along with others, be compound statements (i.e. with `{ }`)?

Reply:
> Shouldn't these, along with others, be compound statements (i.e. with `{ }`)?

Good spotting!
Using the comma operator was an interesting trick by the original author of this tool, apparently to avoid using `{` and `}` here and below.
Never saw this before. I believe that this indirectly violates the more general rule that for readability there should not be more than one C statement per line.

So I've added a respective "don't do" rule to the (partially still open) list of coding style rules in #10725
and fixed all those issues, including the one you mentioned below.

Reply:
> I believe that this indirectly violates the more general rule that for readability there should not be more than one C statement per line.

Well, strictly speaking no.  A comma expression *is* one statement composed of several expressions.

Reply:
I can agree that it's (arguably) not the most readable, though

Reply:
> > I believe that this indirectly violates the more general rule that for readability there should not be more than one C statement per line.
> 
> Well, strictly speaking no. A comma expression _is_ one statement composed of several expressions.

That's why I wrote: "indirectly" and plan to add an explicit "don't do" coding style rule.

Although the two function calls that are combined using the comma operator can be seen as expressions,
their actual use is as statements, and therefore this is against the spirit of not having multiple statements within one line of code.

Reply:
Adapted the newly proposed rule in #10725 to refer to "statement-like expressions".